src/fmu/dataio/scripts/create_case_metadata.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
```

Comment:
Is the __init__.py file needed here?

Reply:
yes, as far as I can tell, it needs to be present for the script to be picked up by ERT.